packages/admin-ui/src/filters/__stories__/filter-status.stories.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react'
+import type { Meta } from '@storybook/react'
+
+import { FilterDisclosure } from '../filter-disclosure'
+import { FilterPopover } from '../filter-popover'
+
+import { FilterFooter } from '../filter-footer'
+import { FilterListbox } from '../filter-listbox'
+import { useFilterMultipleState } from '../filter-multiple/filter-multiple.state'
+import { useFilterState } from '../filter/filter.state'
+
+import { FilterOptionRadio } from '../filter/filter-option-radio'
+
+import { FilterSearchbox } from '../filter-searchbox'
+import { FilterOptionCheckbox } from '../filter-multiple/filter-option-checkbox'
+
+export default {
+  title: 'admin-ui/Filters/status',
+  component: FilterSearchbox,
+} as Meta
+
+// just a sample API
+function api(search: string, delay = 1000): Promise<any[]> {
+  const items = [
+    { country: 'Brazil', id: '1' },
+    { country: 'Bahamas', id: '2' },
+    { country: 'Belarus', id: '3' },
+    { country: 'France', id: '4' },
+    { country: 'Ukraine', id: '5' },
+    { country: 'Australia', id: '6' },
+    { country: 'Afghanistan', id: '7' },
+    { country: 'Albania', id: '8' },
+    { country: 'Algeria', id: '9' },
+    { country: 'American Samoa', id: '10' },
+    { country: 'Andorra', id: '11' },
+    { country: 'Angola', id: '12' },
+    { country: 'Anguilla', id: '13' },
+    { country: 'Antarctica', id: '14' },
+    { country: 'Antigua and Barbuda', id: '15' },
+    { country: 'Argentina', id: '16' },
+    { country: 'Armenia', id: '17' },
+    { country: 'Aruba', id: '18' },
+    { country: 'Austria', id: '19' },
+    { country: 'Azerbaijan', id: '20' },
+  ]
+
+  const res = items.filter((item) =>
+    item.country.toLowerCase().startsWith(search.toLowerCase())
+  )
+
+  return new Promise((resolve) => setTimeout(resolve, delay, res))
+}
+
+export function SingleSearch() {
+  const items = [
+    { label: 'Full', id: '#1' },
+    { label: 'Empty', id: '#2' },
+    { label: 'Half full', id: '#3' },
+    { label: 'Half empty', id: '#4' },
+    { label: 'Unknown', id: '#5' },
+  ]
+
+  const filterState = useFilterState({ fullList: items })
+
+  return (
+    <>
+      <FilterDisclosure state={filterState}>Example</FilterDisclosure>
+      <FilterPopover state={filterState}>
+        <FilterSearchbox id="boxy" />
+        <FilterListbox>
+          {filterState.combobox.matches.map((item) => (
```

Comment:
Exposing the `combobox` nesting and `matches` increases the usage complexity. It would be best if you got rid of this. A call that I expect is something like: 

```suggestion
          {filterState.items.map((item) => (
```

Reply:
Saving it for the next PR